src/genophenocorr/preprocessing/_vep.py

```diff
@@ -134,8 +135,11 @@ def _process_item(self, item: typing.Dict) -> typing.Optional[TranscriptAnnotati
             # if we see a lot of these warnings popping out.
             # Note that Lauren's version of the code had a special branch for missing start, where she set the variable
             # to `1` (1-based coordinate).
-            self._logger.warning('Missing start/end coordinate for %s on protein %s', hgvsc_id, protein_id)
-            protein_effect = None
+            if any(ve in var_effects for ve in SPLICE_EFFECTS):
+                protein_effect = None
```

Comment:
You're repeating yourself here. Ctrl+C Ctrl+V in a small scale. It would be the best to know how to extract the common case so that you do not repeat yourself and the code is, in result, shorter.